src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/Serializer/ProtobufOtlpTagWriter.cs

```diff
@@ -84,6 +84,8 @@ protected override void OnUnsupportedTagDropped(
     internal struct OtlpTagWriterState
     {
         public byte[] Buffer;
+        public int DroppedTagCount;
+        public int TagCount;
```

Comment:
nit: We should update the `ProtobufOtlpTraceSerializer` code to use these too instead of maintaining its own locals for them. Just to keep things consistent.

Reply:
I already got an updated code for it, will open a follow PR to change the `ProtobufOtlpTraceSerializer`